### PR TITLE
feat:display information for gusty wind

### PR DESCRIPTION
**Pull request by sandipa12**
Display the information for gusty weather if the wind speed > 15 mph.
Demo of how the gusty wind is displayed:
Screen Shot 2021-06-29 at 11 18 39 AM

### CODE REVIEW

**Comment by tmsavage** [comment]
Cool feature. In your demo, are you just trying to show what the display looks like for gusty wind even though 7 mph is less than 15 mph?

File: app/lib/clients/weather_client.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Clients::WeatherClient
-  WEATHER_API_URL = 'https://api.openweathermap.org/data/2.5/onecall?lat=%{lat}&lon=%{lon}&appid=%{key}'.freeze
+  WEATHER_API_URL = 'https://api.openweathermap.org/data/2.5/onecall?lat=%{lat}&lon=%{lon}&appid=%{key}&units=imperial'.freeze
```
Comment (cbarber) [inline app/lib/clients/weather_client.rb:2]: Lets split this into a separate commit.

File: app/javascript/components/widgets/weather/current-weather.jsx [hunk 2]
```diff
@@ -32,6 +33,8 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {weather.current.windGust > 15 &&
+        ` , gusty wind at ${weather.current.windGust} mph`}
```
Comment (cbarber) [inline app/javascript/components/widgets/weather/current-weather.jsx:37]: Lets extract this into a component.

Nit: the extra space before the comma seems incorrect.

**Review by cbarber — CHANGES_REQUESTED** [review]

File: app/javascript/components/widgets/weather/current-weather.jsx [hunk 3]
```diff
@@ -32,6 +33,8 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {weather.current.windGust > 15 &&
```
Comment (fallynlogan) [inline app/javascript/components/widgets/weather/current-weather.jsx:36]: What is this comparison doing? Will the wind gust information only be displayed if it is above 15?

Comment (sandipa12) [reply app/javascript/components/widgets/weather/current-weather.jsx:36]: yes, the information is displayed only for the speeed>15

**Comment by sandipa12** [comment]
@tmsavage that's true.

File: app/javascript/components/widgets/weather/current-weather.jsx [hunk 4]
```diff
@@ -32,6 +40,7 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
         <SmallSkyIcon icon={icon} />
       )}
       {description}
+      {Gust(weather)}
```
Comment (allisonlucas) [inline app/javascript/components/widgets/weather/current-weather.jsx:43]: Since Gust is a component, we should include it as such: `<Gust weather={weather} />`. `description` is not a React Component but a string variable, which is why it is included the way it is. It might be helpful to read through the React documentation's overview, which explains React Components and props: https://reactjs.org/tutorial/tutorial.html#overview.

Comment (chewnoill) [reply app/javascript/components/widgets/weather/current-weather.jsx:43]: could be inlined, easiest thing to do I think
```jsx
{ weather.current.windGust > 15 &&
 `, gusty wind at ${weather.current.windGust} mph` }
```

Comment (allisonlucas) [reply app/javascript/components/widgets/weather/current-weather.jsx:43]: Sandipa originally had it inline, but Craig suggested it be pulled out into a component: https://github.com/mojotech/helios2/pull/270#discussion_r660001746

Comment (chewnoill) [reply app/javascript/components/widgets/weather/current-weather.jsx:43]: I'm okay going back to what it was, creating a a whole component for a single span of un-styled text doesn't seem important.

Or if we just renamed the function so it didn't look like it was supposed to be a Component, that just never made it all the way. I imagine if it was named `getWindGustDescription(weather)` I wouldn't bat an eye.

**Review by chewnoill — APPROVED** [review]
🚀

**Review by cbarber — APPROVED** [review]
Nice job! :+1:

File: app/javascript/components/widgets/weather/current-weather.jsx [hunk 5]
```diff
@@ -31,7 +39,7 @@ const CurrentWeather = ({ weather, useLargeIcon }) => {
       ) : (
         <SmallSkyIcon icon={icon} />
       )}
-      {description}
+      <WeatherDescription baseDescription={description} weather={weather} />
```
Comment (allisonlucas) [inline app/javascript/components/widgets/weather/current-weather.jsx:42]: This is great, nice job! 👍
